src/AdminSettings.vue

```diff
@@ -0,0 +1,200 @@
+<!--
+  - @copyright Copyright (c) 2021 Jonas Rittershofer <jotoeri@users.noreply.github.com>
+  -
+  - @author Bjoern Schiessle <bjoern@schiessle.org>
+  - @author Jonas Rittershofer <jotoeri@users.noreply.github.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div id="password-policy" class="section">
```

Comment:
https://nextcloud-vue-components.netlify.app/#/Components/SettingsSection

Reply:
Changed that. 👍 
However, it increases the size of the bundle for that small section from `100KiB` to `330KiB` 🤔 

The SettingsSection is missing the small border at the bottom. Is that a planned new design or a but on components? 🤔

Reply:
> * However, it increases the size of the bundle for that small section from `100KiB` to `330KiB` thinking
> *

We should definitely investigate what weights that much then

Reply:
Hm, seems to be:
- 140K of l10n in component (with another bug, that the single string that wants to use it, is not extracted)
- 30K Iconfonts
- 30K of an additional separate gettext lib

![grafik](https://user-images.githubusercontent.com/47433654/111873820-85536e00-8992-11eb-98ec-c894603d96af.png)
![grafik](https://user-images.githubusercontent.com/47433654/111873844-9c925b80-8992-11eb-900e-b883ae31069b.png)

Reply:
So:
- l10n is tackled on vue-components
- inconfonts need to be separated somehow on vue-components
- gettext/plurals also seems to come from vue-components: if i bundle with a linked local repo, then it is listed within the @nextcloud/vue section.

@skjnldsv Can we resolve this issue here? It's not ideal, but should at least be down to 230 KB bundle-size for now.